From 3de7a65499153dd8f11a38df1968f767dfa5cdaa Mon Sep 17 00:00:00 2001
From: Ray Strode <rstrode@redhat.com>
Date: Tue, 10 Oct 2023 14:35:46 -0400
Subject: [PATCH 06/13] thread: For consistency, s/real_time/realtime/

Most of the code writes "real-time" as "realtime" not "real_time".

The only exception is one function `request_real_time_scheduling`.

This commit changes that function for consistency.
---
 src/backends/native/meta-thread.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/backends/native/meta-thread.c b/src/backends/native/meta-thread.c
index 08d01144d..93a84a8a5 100644
--- a/src/backends/native/meta-thread.c
+++ b/src/backends/native/meta-thread.c
@@ -202,8 +202,8 @@ get_rtkit_property (MetaDBusRealtimeKit1  *rtkit_proxy,
 }
 
 static gboolean
-request_real_time_scheduling (MetaThread  *thread,
-                              GError     **error)
+request_realtime_scheduling (MetaThread  *thread,
+                             GError     **error)
 {
   MetaThreadPrivate *priv = meta_thread_get_instance_private (thread);
   g_autoptr (MetaDBusRealtimeKit1) rtkit_proxy = NULL;
@@ -313,7 +313,7 @@ thread_impl_func (gpointer user_data)
     {
       g_autoptr (GError) error = NULL;
 
-      if (!request_real_time_scheduling (thread, &error))
+      if (!request_realtime_scheduling (thread, &error))
         {
           g_warning ("Failed to make thread '%s' realtime scheduled: %s",
                      priv->name, error->message);
-- 
2.41.0

